lib/Feature/FeatureModelParser.cpp

```diff
@@ -15,6 +15,10 @@ using std::make_unique;
 
 namespace vara::feature {
 
+std::string trim(const std::string &S) {
+  return llvm::StringRef(S).trim().str();
```

Comment:
```suggestion
std::string trim(llvm::StringRef S) {
  return S.trim().str();
```
Would it not be simpler to directly take a string ref if we, in call cases convert it to one anyhow?